### ISSUE TITLE
[BUG] Unable to spectate players/enemies that have been masked/possessed.

### ISSUE THREAD

**Issue by elijah-h-lgtm** [EBro912/SpectateEnemies#4]
Throws and error when trying to spectate enemies, something to do with players who have worn a mask?

```
[Error  : Unity Log] KeyNotFoundException: The given key 'wrthi' was not present in the dictionary.
Stack trace:
System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) (at <787acc3c9a4c471ba7d971300105af24>:IL_001E)
SpectateEnemy.SpectateEnemies.GetNextValidSpectatable () (at <73d98ca278794661a94993b19ce24d8f>:IL_0027)
SpectateEnemy.SpectateEnemies.SpectateNextEnemy () (at <73d98ca278794661a94993b19ce24d8f>:IL_001B)
SpectateEnemy.Patches.PlayerControllerB_Use.Prefix (GameNetcodeStuff.PlayerControllerB __instance) (at <73d98ca278794661a94993b19ce24d8f>:IL_0058)
(wrapper dynamic-method) GameNetcodeStuff.PlayerControllerB.DMD<GameNetcodeStuff.PlayerControllerB::ActivateItem_performed>(GameNetcodeStuff.PlayerControllerB,UnityEngine.InputSystem.InputAction/CallbackContext)
UnityEngine.InputSystem.Utilities.DelegateHelpers.InvokeCallbacksSafe[TValue] (UnityEngine.InputSystem.Utilities.CallbackArray`1[System.Action`1[TValue]]& callbacks, TValue argument, System.String callbackName, System.Object context) (at <69a3ba35f0be43dc9f2827cb84fef646>:IL_001B)
UnityEngine.InputSystem.LowLevel.<>c__DisplayClass7_0:<set_onUpdate>b__0(NativeInputUpdateType, NativeInputEventBuffer*)
UnityEngineInternal.Input.NativeInputSystem:NotifyUpdate(NativeInputUpdateType, IntPtr)

[Error  : Unity Log] KeyNotFoundException while executing 'performed' callbacks of 'Movement/ActivateItem[/Mouse/leftButton]'
```

**Comment by EBro912** [comment]
I know exactly why that happens, thank you for your report. That's totally my fault, I'll release a fix for it as soon as I can.

**Comment by EBro912** [comment]
Should be fixed as of v2.1, make another issue if the bug persists.
